tools/CreateMappings_rules.json

```diff
@@ -46,7 +46,8 @@
     { "regex": "DataFactories", "group": "Data Factories", "alias": "Data Factories" },
 
     { "regex": "Storage", "alias": "Storage" },
-
+    { "regex": "StorageSync", "alias": "StorageSync" },
```

Comment:
@ankushbindlish2 to be safe, I would swap this line with the `Storage` line above so the `StorageSync` cmdlets are processed first and then the `Storage` cmdlets -- my fear is that the change as it is would still cause the `StorageSync` cmdlets to be filed under the `Storage` header since they would match the `Storage` regex first